lex

lex-neufeld opened the pull request.

lex-neufeld committed:
class Display added
Trivia.answers changed to a list of pointers to Trivia objects
importList() updated to set Trivia.display and Trivia.answers
MemGlobeApplication.java updated to test new features

lex-neufeld committed:
importList() is way fancier now. Requires a #Key in the import file to work
added countriesAndAreaAndPopulation.txt which has a #Key and more Trivia
However, it creates a currentSet that doesn't actually make sense as flashcards... progress?

CarinaHolmes commented on src/main/java/CardSets.java:
```diff
@@ -4,42 +4,58 @@
+                //set the answers for each Trivia to the other items
+                for (int i = 1; i < currentLine.length; i++){
+                    for (int o = 1; o < currentLine.length; o++) {
+                        if (i == o) {
```
why?

CarinaHolmes commented on src/main/java/MemGlobeApplication.java:
```diff
@@ -5,16 +5,20 @@ public class MemGlobeApplication {
-        System.out.println(currentSet.get(11).id);
-        System.out.println(currentSet.get(11).answers);
+
+        for (int i = 0; i<6;i++){
```
Why are you limiting your loop to 6 counts? Wouldn't is make more sense to loop through the whole currentSet?

lex-neufeld committed:
countriesAndCapitalsAndCurrency.txt added. This should be easy to work with but suitably complex for development.

lex-neufeld committed:
renamed latest latest source file
importAdda247() is largely duplicate code, but contains improvements not copied to importList() (which supports the previous source file). This is going to get messy, but that is probably just the way it will have to be with each source having its own import method.
